build.gradle

```diff
@@ -310,7 +316,7 @@ configure(subprojects.findAll { it.name != 'util' && it.name != 'mongo-java-driv
 
 task docs(type: Javadoc) {
     source subprojects.grep({ it.name != 'util' }).collect {project -> project.sourceSets.main.allJava }
-    options = subprojects.first().javadoc.options
+//    options = subprojects.first().javadoc.options
```

Comment:
I tested this and I don't think this line is necessary at all.  The javadoc settings on lines 105-150 are picked up for all the projects whether or not this line is here.